torch/onnx/_internal/fx/torch_export_graph_extractor.py

```diff
@@ -42,6 +42,9 @@ def generate_fx(
         #         args=model_args,  # type: ignore[arg-type]
         #         kwargs=model_kwargs,  # type: ignore[arg-type]
         #     )
+        # TODO: input_adapter and output_adapter should be based on pytree flatten/unflatten API?
```

Comment:
Unrelated outdated comment?

Reply:
nope, an actual extension of the comments above. We probably don't need to use the input adapter steps we created ro `ExportedProgram` for each change (flatten, remove none, etc). A simple inputAdfapterstep that calls the pytree to unflatten input and another to unflatten output seems more correct. The torch.export contract is based on "pytreeable" input/output

Reply:
The code today is the same as you mentioned. `FlattenInputWithTreeSpecValidationInputStep` is the pytree flatten step. If you want you can rename it to put pytree in the name. 

Or maybe you meant `ExportedProgram` needs only this step? That I don't think so since other steps are a requirement from elsewhere, e.g., onnx.

Reply:
`FlattenInputWithTreeSpecValidationInputStep  uses pytree, that is what I meant. great, no renaming needed